aiida_vasp/assistant/parameters.py

```diff
@@ -12,22 +12,7 @@
 from aiida.plugins import DataFactory
 from aiida_vasp.utils.extended_dicts import update_nested_dict
 
-FUNCTIONAL_PARAMETERS = {
-    'charge': {
-        'wave': True,
-        'charge': True,
-        'potential': True,
-        'constant_charge': True,
-        'constant_atomic': True
-    },
-    'smearing': {
-        'mp': True,
-        'gaussian': True,
-        'fermi': True,
-        'partial': True,
-        'tetra': True
-    }
-}
+_ALLOWED_NAMESPACES = ['electronic', 'smearing', 'charge', 'dynamics', 'bands', 'relax', 'converge', 'code']
```

Comment:
For backward compatibility, I would suggest we also add `vasp` as the allowed namespaces. 
Those from both `code` and `vasp` can be combined together for messaging use.